api_mongoDB.go

```diff
@@ -102,6 +106,181 @@ func GetUniqueIdentity() int32 {
 	}
 }
 
+func GetUniqueIdentityWithinRange(min int32, max int32) int32 {
+	rangeCollection := Client.Database(dbName).Collection("range")
+
+	rangeFilter := bson.M{}
+	rangeFilter["_id"] = "uniqueIdentity"
+
+	for {
+		count := rangeCollection.FindOneAndUpdate(context.TODO(), rangeFilter, bson.M{"$inc": bson.M{"count": 1}})
+
+		if count.Err() != nil {
+			counterData := bson.M{}
+			counterData["count"] = min
+			counterData["_id"] = "uniqueIdentity"
```

Comment:
isnt it looks _id is special key and it will cause only 1 document with _id = uniqueIdentity ?